pom.xml

```diff
@@ -121,6 +121,7 @@
                  <failsOnError>true</failsOnError>
                  <linkXRef>false</linkXRef>
                  <useFile/>
+                 <includeTestSourceDirectory>true</includeTestSourceDirectory>
```

Comment:
i just havent gotten gradle to work with our nexus yet unfortunately :/
ive played with it a bit, but it doesnt work. i need to try to get hold of someone that knows nexus a bit better than i do.